Make these inline definitions "extern inline" to keep a copy of
the code in the object files.

--- linux-sunxi-d47d367036be38c5180632ec8a3ad169a4593a88/drivers/input/mouse/lifebook.h	2015-03-15 20:30:43.000000000 +0100
+++ linux-sunxi-d47d367036be38c5180632ec8a3ad169a4593a88/drivers/input/mouse/lifebook.h	2016-12-11 23:57:26.116108248 +0100
@@ -16,14 +16,14 @@
 int lifebook_detect(struct psmouse *psmouse, bool set_properties);
 int lifebook_init(struct psmouse *psmouse);
 #else
-inline void lifebook_module_init(void)
+extern inline void lifebook_module_init(void)
 {
 }
-inline int lifebook_detect(struct psmouse *psmouse, bool set_properties)
+extern inline int lifebook_detect(struct psmouse *psmouse, bool set_properties)
 {
 	return -ENOSYS;
 }
-inline int lifebook_init(struct psmouse *psmouse)
+extern inline int lifebook_init(struct psmouse *psmouse)
 {
 	return -ENOSYS;
 }
